build_scripts/build_usd.py

```diff
@@ -2160,9 +2176,9 @@ def ForceBuildDependency(self, dep):
 if which("cmake"):
     # Check cmake requirements
     if Windows():
-        # Windows build depend on boost 1.70, which is not supported before
-        # cmake version 3.14
-        cmake_required_version = (3, 14)
```

Comment:
Requiring newest cmake while not force using 1.78 seems unreasonable. A user would have to install cmake 3.24 for all windows build, even with VS2019 or VS2017